src/Sylius/Behat/Context/Transform/CountryContext.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Addressing\Converter\CountryNameConverterInterface;
+use Sylius\Component\Addressing\Model\CountryInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+final class CountryContext implements Context
+{
+    /**
+     * @var RepositoryInterface
+     */
+    private $countryRepository;
+
+    /**
+     * @var FactoryInterface
+     */
+    private $countryFactory;
+
+    /**
+     * @var CountryNameConverterInterface
+     */
+    private $countryNameConverter;
+
+    /**
+     * @param RepositoryInterface $countryRepository
+     * @param FactoryInterface $countryFactory
+     * @param CountryNameConverterInterface $countryNameConverter
+     */
+    public function __construct(
+        RepositoryInterface $countryRepository,
+        FactoryInterface $countryFactory,
+        CountryNameConverterInterface $countryNameConverter
+    ) {
+        $this->countryRepository = $countryRepository;
+        $this->countryFactory = $countryFactory;
+        $this->countryNameConverter = $countryNameConverter;
+    }
+
+    /**
+     * @Transform /^country "([^"]+)"$/
+     */
+    public function getCountryByName($countryName)
+    {
+        $countryCode = $this->countryNameConverter->convertToCode($countryName);
+        $country = $this->countryRepository->findOneBy(['code' => $countryCode]);
+
+        if (null === $country) {
+            throw new \InvalidArgumentException(sprintf('Country with name %s does not exist', $countryName));
+        }
+
+        return $country;
+    }
+
+    /**
+     * @Transform /^"([^"]*)" country enabled$/
```

Comment:
IMHO we shouldn't allow for these constructs, creating entities in transformer is controversial, but setting or toggling another property on it is even more. Anyway, `/^country "([^"]+)"$/` transformer does not create a country which is quite inconsistent.